operator/internal/manifests/gateway_tenants.go

```diff
@@ -140,7 +140,14 @@ func ConfigureOptionsForMode(cfg *config.Options, opt Options) error {
 		return nil // nothing to configure
 	case lokiv1.OpenshiftLogging, lokiv1.OpenshiftNetwork:
 		if opt.OpenShiftOptions.BuildOpts.AlertManagerEnabled {
-			return openshift.ConfigureOptions(cfg)
+			monitorServerName := fqdn(openshift.MonitoringSVCUserWorkload, openshift.MonitoringUserwWrkloadNS)
+			return openshift.ConfigureOptions(
+				cfg,
+				opt.OpenShiftOptions.BuildOpts.UserWorkloadAlertManagerEnabled,
+				BearerTokenFile,
+				alertmanagerUpstreamCAPath(),
+				monitorServerName,
+			)
 		}
 		return nil
```

Comment:
This requires to be split into two cases now, one per tenant mode. I.e. mode `openshift-logging` is the only one handling the UWM alertmanager override.